client/src/constants.ts

```diff
@@ -56,10 +56,17 @@ export const CONTRACT_ADDRESSES = {
   1: {
     REN_TOKEN: '0x408e41876cCCDC0F92210600ef50372656052a38',
     DARKNODE_REGISTRY: '0x2D7b6C95aFeFFa50C068D50f89C5C0014e054f0A',
+    REN_POOL: '0x0000000000000000000000000000000000000000',
   },
   42: {
     REN_TOKEN: '0x2CD647668494c1B15743AB283A0f980d90a87394',
     DARKNODE_REGISTRY: '0x9954C9F839b31E82bc9CA98F234313112D269712',
```

Comment:
Here I see we repeating ourselves with addresses configuration. We already have that configuration in Hardhat, we need to undertand the best way to reuse from both projects.

Reply:
Agree. I'll give it some thought. The only address that changes (but still we should be able to store it somewhere at deploy time) is the RenPool address.

Reply:
In PR #88 I tried to reuse contract address from `client`.